watcher/lib/influxdb/rollups.go

```diff
@@ -34,35 +34,40 @@ type Rollup struct {
 	// Retention is the retention policy for this rollup
 	Retention string `json:"retention"`
 	// Measurement is the name of the measurement to run rollup on
-	Measurement string `json:"measurement"`
+	Measurement string `json:"measurement,omitempty"`
 	// Name is both the name of the rollup query and the name of the
 	// new measurement rollup data will be inserted into
 	Name string `json:"name"`
 	// Functions is a list of functions for rollup calculation
 	Functions []Function `json:"functions"`
+	// CustomFrom is a custom 'from' clause
```

Comment:
Given the behavior of Check, I'd mention that `CustomFrom` complements `Measurement` - i.e. one of them must be provided.